crates/nu-protocol/src/debugger/profiler.rs

```diff
@@ -277,6 +307,16 @@ fn collect_data(
         "parent_id" => Value::int(parent_id.0 as i64, profiler_span),
     };
 
+    if profiler.collect_lines {
+        if let Some((fname, line_num)) = find_file_of_span(engine_state, element.element_span) {
+            row.push("file", Value::string(fname, profiler_span));
+            row.push("line_num", Value::int(line_num as i64, profiler_span));
```

Comment:
Sorry I missed this spot
```suggestion
            row.push("line", Value::int(line_num as i64, profiler_span));
```